tests/unit/wc-payment-api/test-class-wc-payments-api-client.php

```diff
@@ -1987,6 +1987,99 @@ public function test_get_document_error() {
 		$this->payments_api_client->get_document( 'someDocument' );
 	}
 
+	/**
+	 * Test a successful fetch of a single authorization.
+	 *
+	 * @throws Exception In case of test failure.
+	 */
+	public function test_get_authorization_success() {
+		$payment_intent_id = 'pi_123smtm';
+
+		$this->set_http_mock_response(
+			200,
+			[
+				'payment_intent_id' => $payment_intent_id,
+			]
+		);
+
+		$authorization = $this->payments_api_client->get_authorization( $payment_intent_id );
+		$this->assertEquals( $payment_intent_id, $authorization['payment_intent_id'] );
+	}
+
+	/**
+	 * Test fetching of non existing authorization.
+	 *
+	 * @throws Exception In case of test failure.
+	 */
+	public function test_get_authorization_not_found() {
+		$payment_intent_id = 'pi_123smtm';
+		$error_code        = 'authorization_missing';
+		$error_message     = 'The authorization you asked for does not exist';
+
+		$this->set_http_mock_response(
+			404,
+			[
+				'error' => [
+					'code'    => $error_code,
```

Comment:
I think we can inline the value here.

Reply:
Indeed, changed here f88b5cfd I left the other ones as variables given they are used below in the test